src/core/commands.ts

```diff
@@ -570,11 +570,6 @@ export const removeDestination = async (payload: {
     project.videoApi.project.destinations.filter(
       (x) => x.destinationId !== destinationId,
     )
-
-  trigger('DestinationRemoved', {
-    projectId,
-    destinationId,
-  })
```

Comment:
Event will trigger when the LiveApiEvent actually says so. Removing this to prevent event from triggering twice.